core/src/components/textarea/test/label-placement/textarea.e2e.ts

```diff
@@ -234,3 +211,32 @@ configs().forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('textarea: label overflow'), () => {
+    test('label property should be truncated with ellipses', async ({ page }) => {
+      await page.setContent(
+        `
+            <ion-textarea label="Label Label Label Label Label" placeholder="Text Input"></ion-textarea>
+          `,
+        config
+      );
+
+      const textarea = page.locator('ion-textarea');
+      expect(await textarea.screenshot()).toMatchSnapshot(screenshot(`textarea-label-truncate`));
+    });
+    test('label slot should be truncated with ellipses', async ({ page }) => {
```

Comment:
```suggestion
    test('label slot should be truncated with an ellipsis', async ({ page }) => {
```

Nitpick, feel free to ignore but singular is ellipsis